Newsletters: add 298 (2024-04-17)

harding opened the pull request:
- [x] Lede, releases/RCs, topic entries @harding 
- [x] Client/Services section @bitschmidty 

~~I'll check later in the week/weekend to look again for possible news stories or PR merges, but I'm guessing a lot of people are busy with the conference, so it might be a slow news week.~~

bitschmidty commented:
Might consider: https://delvingbitcoin.org/t/research-into-the-effects-of-a-cluster-size-limited-mempool-in-2023/794

murchandamus reviewed (commented):
LGTM so far. I have no improvement suggestions at this time.

Edit:
Looks like 27.0 has been tagged: https://github.com/bitcoin/bitcoin/releases/tag/v27.0

harding commented:
Added lede, releases/RCs, and topic entries.  Added FIXME for @bitschmidty to remove the Bitcoin Core 27 release announcement if the binaries haven't been uploaded to BitcoinCore.org by Wednesday morning.  Thanks @bitschmidty and @murchandamus for the reviews!

murchandamus commented on _posts/en/newsletters/2024-04-17-newsletter.md:
```diff
@@ -92,10 +96,37 @@ FIXME:bitschmidty
+  For a complete list of major changes, please see the [release
+  notes][bcc27 rn].
+
+- [BTCPay Server 1.13.1][] is the is the latest release of this
```
```suggestion
- [BTCPay Server 1.13.1][] is the latest release of this
```

bitschmidty commented on _posts/en/newsletters/2024-04-17-newsletter.md:
```diff
@@ -0,0 +1,204 @@
+  recorded every transaction his node received in 2023 and has now run
+  them through a development version of Bitcoin Core with [cluster
+  mempool][topic cluster mempool] enabled to quantify differences
+  between the two systems.  Some of his findings include:
```
It may not be clear what the two systems are here

murchandamus commented:
27.0 released: https://github.com/bitcoin/bitcoin/releases/tag/v27.0

harding commented:
Pushed edits for @murchandamus and @bitschmidty review comments (thanks!) and reviewed client/services section (looks great, thanks @bitschmidty !).

bitschmidty reviewed (approved):
Updates look good, took the suggestion from @LarryRuane, squashed
